librz/bin/format/coff/coff_reloc.c

```diff
@@ -97,66 +134,42 @@ static void relocs_foreach(struct rz_bin_coff_obj *bin, RelocsForeachCb cb, void
 			ut8 patch_buf[8];
 			if (sym_vaddr) {
 				switch (bin->hdr.f_magic) {
+				// TODO: Missing handling of MIPS architecture
 				case COFF_FILE_MACHINE_I386:
 					switch (rel[j].rz_type) {
 					case COFF_REL_I386_DIR32:
```

Comment:
create a function also for this.

Reply:
a function for what?

Reply:
Yes, for the whole `switch (rel[j].rz_type)`